packages/core/src/Drawer/Drawer.tsx

```diff
@@ -14,6 +14,7 @@ export type PlacementOptions = 'top' | 'bottom' | 'right' | 'left'
 export type DrawerProps = SpaceProps &
   LayoutProps & {
     children?: string | React.ReactNode
+    customHeader?: string | React.ReactNode
     heading?: string | React.ReactNode
```

Comment:
IMHO, `customHeader` and `heading` both allowing a `ReactNode` is going to create confusion